parachain/pallets/erc20-app/src/payload.rs

```diff
@@ -5,6 +5,15 @@ use codec::Encode;
 use ethabi::{self, Token};
 use artemis_ethereum::{H160, U256};
 
+// Message from Ethereum (SCALE-encoded)
```

Comment:
should this be in this PR?

Reply:
No; generally speaking, this PR shouldn't touch anything outside the `ethereum` folder. This must have been changed in master at some point and this legacy code block got orphaned in this branch. I removed it in https://github.com/Snowfork/polkadot-ethereum/pull/266/commits/a23900ee7e7063222e39bb4ae4ef65ef579cee6b so that the pallet matches what's in master.